src/renderer/api/endpoints/resource-applier.api.ts

```diff
@@ -48,7 +48,7 @@ export const resourceApplierApi = {
           }
         });
 
-        return items.length === 1 ? items[0] : items;
+        return items[0] as KO ?? null;
```

Comment:
This changes behaviour but I have no idea why we have returned an item or an array based on length 🤔

Reply:
I agree that this changes behaviour, but if you look at the declared types. We never said we were returning an array. So I don't believe anyone was expecting an array.

Reply:
Yeah, I agree.. old behaviour is a bit confusing.